packages/audits/src/components/AuditsTableCell.tsx

```diff
@@ -18,6 +20,29 @@ const AuditsTag: CellComponent = (props) => {
   );
 };
 
+// Logos take from the cdnjs site. The repository with the src is here https://github.com/alrra/browser-logos
+const UserAgent: CellComponent = ({ value }) => {
+  const browser = useMemo(() => {
+    const browser = new UAParser(value).getBrowser().name;
+    return browser && browsers.includes(browser.toLowerCase() ?? '') ? browser : 'Web';
```

Comment:
- browser.toLowerCase() does not return undefined then the ?? '' will not work 
- what if the UAParser failed to parse the value? 
- I guess the getBrowser function return nullable value then the .name will throw exception

Reply:
Yes, the getBrowser function returns a nullable value. I'll fix it.